pages/store/claim.vue

```diff
@@ -69,6 +69,29 @@ const cartId = computed(() => getRouteQuery('cart_id'))
 const claimingToken = computed(() => getRouteQuery('claiming_token'))
 const paymentId = computed(() => getRouteQuery('payment_id'))
 
+const loadingLabels = [
+  $t('claim_page_loading_step_1'),
+  $t('claim_page_loading_step_2'),
+  $t('claim_page_loading_step_3'),
+]
```

Comment:
Consider having an array different texts in each step, so that we can still loop with different texts when we are stuck at a particular step

Reply:
[🚸 Enhance loading labels with additional variants](https://github.com/likecoin/3ook-com/pull/269/commits/2e4e7cded851393ea3c9bd005b60a7056d98c064)